core/server/data/meta/schema.js

```diff
@@ -38,15 +38,15 @@ function trimSchema(schema) {
 function trimSameAs(data, context) {
     var sameAs = [];
 
-    if (context === 'post') {
-        if (data.post.primary_author.website) {
-            sameAs.push(escapeExpression(data.post.primary_author.website));
+    if (context === 'post' || context === 'page') {
```

Comment:
This commit refs https://github.com/TryGhost/Ghost/issues/10082.
I'd suggest to add a bug fix emoji and a ref to this issue as part of the commit :)

Reply:
Good call. Will do :+1: